coral-hive/src/main/java/com/linkedin/coral/transformers/CoralRelToSqlNodeConverter.java

```diff
@@ -348,6 +358,27 @@ public Result visit(Uncollect e) {
         ImmutableMap.of(projectResult.neededAlias, e.getRowType()));
   }
 
+  /**
+   * Override this method to avoid the duplicated alias for {@link org.apache.calcite.rel.logical.LogicalValues}.
+   * So that for the input SQL like `SELECT 1`, the translated SQL will be like:
+   *
+   * SELECT 1
+   * FROM (VALUES  (0)) t (ZERO)
+   *
+   * Without this override, the translated SQL contains duplicated alias `t`:
+   *
+   * SELECT 1
+   * FROM (VALUES  (0)) t (ZERO) t
+   *
+   * which is wrong.
+   */
+  @Override
+  public Result visit(Values e) {
```

Comment:
this method accepts a relNode and generates a 'Result' object. Could you update the documentation to reflect the motivation for the override regarding input & output? Please refer to the other override documentation for reference.

Reply:
@ljfgem maybe explain how the new `new Result()` achieves your goal?

Reply:
@wmoustafa Done, PTAL.